### DISCUSSION TITLE
Update the dependency maennchen/zipstream-php to the latest version

### DISCUSSION THREAD

**Discussion by cameronjohnson-mz** [PHPOffice/PhpSpreadsheet#3536] [Ideas]
Composer is saying this package is using an outdated version of ` maennchen/zipstream-php`. 
Can the next release include the latest version 3.0.2?

**Comment by oleibman** [comment]
PR #3410 is designed to do this. My plan is to merge it after the next release, which is expected this week. I think it needs a bit more testing before being part of a release. Closing this issue as a duplicate of issue #3430.
